### DISCUSSION TITLE
ChatGPT Plugins

### DISCUSSION THREAD

**Discussion by lucasjans** [ztjhz/BetterChatGPT#263] [Ideas]
Are there any plans to add support for ChatGPT Plugins?
https://openai.com/blog/chatgpt-plugins

Specifically, I would love to add web browsing capabilities, or extend this tool to work with my business systems in a secure way.

**Comment by ztjhz** [comment]
We are planning to add support for plugins!

**Reply by Jarrodsz** [reply]
Can we have a tenplate for writing custum plugins in local dev?

**Comment by Jarrodsz** [comment]
Any updates on this?

**Comment by othmanelhoufi** [comment]
Developers are successfully adding plug-ins in this Chat UI App: https://github.com/dotneet/smart-chatbot-ui
I would love to see the same solution implemented in BetterChatGPT  😊
